### DISCUSSION TITLE
Git auto-deploy stopped working.

### DISCUSSION THREAD

**Discussion by ceiphr** [vercel/community#2947] [Help] [answered]
### Summary

I'm pushing changes to my repo, but no deployments are triggered on Vercel. The repo is on GitHub. I tried reconnecting it to my Vercel Project from `Project Settings`.

I saw this issue that says it was an incident a few weeks ago (https://github.com/orgs/vercel/discussions/2777), but this started happening for me maybe three days ago. It looks like both GitHub (https://www.githubstatus.com/) and Vercel (https://www.vercel-status.com/) are reporting no issues.

For the time being I'm deploying from the Vercel CLI as a work around.

### Steps to Reproduce

1. Push a commit to the main branch of the connected repo.
2. See there is no deployed triggered for the project in Vercel's dashboard.

**Comment by amyegan** [comment]
Do you happen to have a `vercel.json` config file with `git.deploymentEnabled` set? Or something in the Ignored Build Step field in `Project Settings > Git`?

**Comment by ceiphr** [comment]
I have a serverless Go function in `/api`, so I added these fields to my `vercel.json`:

```json
{
    "version": 2,
    "framework": "nextjs",
    "build": {
        "use": "@vercel/go@latest",
        "src": "api/search.go",
        "config": {
            "zeroConfig": true
        }
    },
    "functions": {
        "api/search.go": {
            "includeFiles": "content/posts/**"
        }
    },
}
```

I don't have any `git.deploymentEnabled` field. Nothing in `Ignored Build Step` either.

Could this be related to having the Go function in `/api` and some TypeScript endpoints in `pages/api`? I was struggling to find documentation on how to use both or if they'd conflict.

**Comment by ceiphr** [comment, marked as answer]
It was related to my use of Go. I don't know why, but after removed that stuff from my config and deleting my `/api` folder, everything returned to normal.
